src/contact.ts

```diff
@@ -232,6 +234,10 @@ export class Contact implements Sayable {
       query = { name: /.*/ }
     }
 
+    if (query[0] === 'remark') {
+      log.warn('Contact', 'findAll(remark:%s) DEPRECATED, use findAll(alias:%s) instead.')
```

Comment:
no need to add `query[0] = 'alias'`, because I had deal the alias in the following code:
```
const keyMap = {
      name:   'NickName',
      // should be deprecated
      remark: 'RemarkName',
      alias:  'RemarkName',
    }
...
 if (filterValue instanceof RegExp) {
      filterFunction = `(function (c) { return ${filterValue.toString()}.test(c.${filterKey}) })`
    } else if (typeof filterValue === 'string') {
      filterValue = filterValue.replace(/'/g, '\\\'')
      filterFunction = `(function (c) { return c.${filterKey} === '${filterValue}' })`
    } else {
      throw new Error('unsupport name type')
    }
```

If you are sure to deprecate function `Contact.find(remark:string)` right now, I will delete this